ckanext/xloader/tests/test_loader.py

```diff
@@ -777,11 +760,7 @@ def test_kml(self):
                 mimetype="text/csv",
                 logger=logger,
             )
-        assert "Error with field definition" in str(exception.value)
-        assert (
-            '"<?xml version="1.0" encoding="utf-8" ?>" is not a valid field name'
-            in str(exception.value)
-        )
+        assert "Tabulator error: Format \"kml\" is not supported" in str(exception.value)
```

Comment:
This should probably use six.

Reply:
We already have `from six import text_type as str` in this file, so I think this is OK like so.